dags/churn.py

```diff
@@ -24,3 +24,14 @@
                       uri="https://raw.githubusercontent.com/mozilla/mozilla-reports/master/etl/churn.kp/orig_src/Churn.ipynb",
                       output_visibility="public",
                       dag=dag)
+
+t1 = EMRSparkOperator(task_id="churn_to_csv",
+                      job_name="Generate Churn CSV files",
+                      execution_timeout=timedelta(hours=4),
+                      instance_count=1,
+                      env={"date": DS_WEEKLY},
+                      uri="https://raw.githubusercontent.com/mozilla/mozilla-reports/master/etl/churn_to_csv.kp/orig_src/churn_to_csv.ipynb",
+                      output_visibility="public",
```

Comment:
I didn't know that, I'll keep it in mind for other jobs that collect to local disk. :+1: